### PR TITLE
fix: ext-12-fix-testnet-smart-contracts

### PR DESCRIPTION
**Pull request by giovannifranchi**
Changes implemented:

- The main root of error lies in the fact that the gnosis multisig in not registered as a client in the Supra Deposit contract. This process can only be done by the client trough the following [form](https://supra.com/docs/vrf/vrf-subscription-model/)

- The other problem related to the after deployment scripts for Blast and Base was the fact that the script was using safeBatch package which does not support the target chains. 

batch-script

- The script is now creating a pending transaction in the gnosis multisig wallet that needs to be then confirmed individually by the other signers.

- Additionally a code snippet for executing the whole transactions has been added. Note this code can only work if all signers PKs are provided in the .env file

### CODE REVIEW

File: script/Base/post-deployment/01_configureVRFSubscription.s.sol [hunk 1]
```diff
@@ -36,21 +40,67 @@ contract ConfigureVRFSubscription_Base is BatchScript {
             perpetualMint
         );
 
-        addToBatch(supraVRFDepositContract, addContractToWhitelistTx);
+        // This version is makes possible to directly make the gnosis safe execute the transactions required by broadcasting the transactions for each signer.
```
Comment (0xinsi) [inline script/Base/post-deployment/01_configureVRFSubscription.s.sol:43]: There will never be an instance where a single `.env` file holds multiple multisig signer PKs - it seems this is in stark contrast to general security practices.

Please remove the commented code.

File: script/Blast/post-deployment/02_configureVRFSubscription.s.sol [hunk 2]
```diff
@@ -36,21 +40,69 @@ contract ConfigureVRFSubscription_Blast is BatchScript {
             perpetualMint
         );
 
-        addToBatch(supraVRFDepositContract, addContractToWhitelistTx);
+        // This version is makes possible to directly make the gnosis safe execute the transactions required by broadcasting the transactions for each signer.
```
Comment (0xinsi) [inline script/Blast/post-deployment/02_configureVRFSubscription.s.sol:43]: Similar as above.

**Review by 0xinsi — CHANGES_REQUESTED** [review]
Generally looks good.

Please ensure `make prettier` is run - it seems that the formatting is different between two almost identical files.

Has this been run on a testnet and verified?

Additionally, do you reckon it makes sense that another script be added which just sets values on testnets (ie tenderly testnets) rather than submitting txs?

**Comment by giovannifranchi** [comment]
They cannot be tested because as stated in the previous comment, Supra client whitelisting is centralised and needs to pass trough a form that gets sent to the Supra team. This needs to be done by the protocol owner. Furthermore developing that kind of script would not provide in my opinion any type of insight on the side of contract testing. Doing such a thing would just take a snapshot of the blockchain at that moment but will not be able to perform scenarios related to the Supra service since most of that part happens off-chain

**Pull request closed without merging**